pkg/apis/registries/v1alpha1/repository_types.go

```diff
@@ -2,16 +2,23 @@ package v1alpha1
 
 import (
 	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
> This import section ain't big enough for the both of us :gun:

I'd suggest to stick with `corev1` for the alias as in other packages:
```suggestion
	corev1 "k8s.io/api/core/v1"

```

Reply:
done